src/main/java/com/example/provider/json/SerializedVehicle.java

```diff
@@ -39,6 +42,12 @@ abstract class SerializedVehicle {
   abstract int maximumCapacity();
   /** Supported trip types */
   abstract List<TripType> supportedTripTypes();
+  /** Last location of vehicle */
+  abstract SerializedLastLocation lastLocation();
+  /** Vehicle attributes */
+  abstract List<VehicleAttribute> vehicleAttributes();
```

Comment:
Nit: is the intent to serialize the whole list of 'attributes' vs just 'restaurantId'?

Reply:
For generalization purposes, I think 'resuaurantID' will be added as an attribute instead of a separate vehicle field. Attributes is a required field for 'GMTSVehicle', so we are serializing the whole list.